app/src/main/java/org/asteroidos/sync/ble/NotificationService.java

```diff
@@ -72,19 +74,32 @@ public void onReceive(Context context, Intent intent) {
             String event = intent.getStringExtra("event");
             if (Objects.equals(event, "posted")) {
                 String packageName = intent.getStringExtra("packageName");
+                NotificationPreferences.NotificationOption notificationOption =
+                        NotificationPreferences.getNotificationPreferenceForApp(context, packageName);
+                if (notificationOption == NotificationPreferences.NotificationOption.NO_NOTIFICATIONS)
+                    return;
+
                 int id = intent.getIntExtra("id", 0);
                 String appName = intent.getStringExtra("appName");
                 String appIcon = intent.getStringExtra("appIcon");
                 String summary = intent.getStringExtra("summary");
                 String body = intent.getStringExtra("body");
+                String vibration;
+                if (notificationOption == null
+                        || notificationOption == NotificationPreferences.NotificationOption.DEFAULT
```

Comment:
I don't know. It's a personal preference that should anyways be made user configurable ASAP. I'll change the default to normal vibration in code for now until it's made configurable.